arduino-ide-extension/src/browser/widgets/cloud-sketchbook/cloud-sketchbook-tree-widget.tsx

```diff
@@ -61,10 +61,10 @@ export class CloudSketchbookTreeWidget extends SketchbookTreeWidget {
           </div>
         </div>
         <button
-          className="theia-button"
+          className="theia-button uppercase"
           onClick={() => shell.openExternal('https://create.arduino.cc/editor')}
         >
-          {nls.localize('cloud/GoToCloud', 'GO TO CLOUD')}
+          {nls.localize('arduino/cloud/goToCloud', 'Go to Cloud')}
```

Comment:
Changing the localization key from `'cloud/GoToCloud'` to `'arduino/cloud/goToCloud'` is right, but what about all the translations of this string? Like this we would lose them all.

Reply:
I have also thought about the impact of prior changes to the keys. It was previously discussed here:

https://github.com/arduino/arduino-ide/issues/1431#issuecomment-1248000820

I think there is an impact, but it can't quite be described as "lose them" because the translations are still stored in [the "translation memory"](https://docs.transifex.com/translation-memory/tm-introduction) on Transifex. So they don't need to be translated over again from English, but I don't think this happens automatically, meaning there is still an impact. I believe the translators will still need to select the strings with changed keys and then accept the auto-completed translation offered by Transifex.

Reply:
So it seems to me that we should avoid frivolous key changes, but also shouldn't feel locked into the existing key names in cases where there is a significant benefit to restructuring them.